### ISSUE TITLE
Progress Bar Chapter Mark Visibility

### ISSUE THREAD

**Issue by offset-torque** [koreader/koreader#8191]
Chapter marks in the progress bar becomes nearly invisible when their background is gray. 

Especially when reading in low light conditions on an e-ink screen (which already has lower contrast ratio because blacks are closer to dark gray) it is hard to check how far you are in a chapter with a glance.

 If their color can be inverted when the chapters are "read", contrast and visibility will be much better as indicated below:

![progress bar inverted](https://user-images.githubusercontent.com/84420582/132461157-0c7486a9-740b-45ad-8f58-4e0faf648054.png)

**Comment by hius07** [comment]
I like the idea.

Before

<kbd>![1](https://user-images.githubusercontent.com/62179190/132481301-e15a81fa-e06c-4cc9-b974-e211f9af6964.png)</kbd>

After

<kbd>![2](https://user-images.githubusercontent.com/62179190/132481335-c0139801-4b27-4e14-9fca-68acd113d5a8.png)</kbd>

**Comment by poire-z** [comment]
I'm fine with the idea, if it looks not too washed out on eInk.

**Comment by poire-z** [comment]
Some issue when it is really crowded, see https://github.com/koreader/koreader/pull/8195#issuecomment-915172922.
